### PR TITLE
Wolfram

### PR DESCRIPTION
**Pull request by wdecker**

### CODE REVIEW

File: src/Modules/UngradedModules.jl [hunk 1]
```diff
@@ -826,12 +826,12 @@ end
...
 
 Given a vector `V` of rank `F` elements of `G`, 
-return the homomorphism `F` $\to$ `G` defined by sending `F[i]` to `V[i]`.
+return the homomorphism  defined by sending `F[i]` to `V[i]`.
 """
 hom(F::FreeMod, G::ModuleFP, V::Vector) = FreeModuleHom(F, G, V)
 
 @doc Markdown.doc"""
-    hom(F::FreeMod{T}, G::ModuleFP{T}, A::MatElem{T}) where T
+    hom(F::FreeMod{T}, G::FreeMod{T}, A::MatElem{T}) where T
```
Comment (AlexD97) [inline src/Modules/UngradedModules.jl]: May I ask why this change was necessary?

Comment (wdecker) [reply src/Modules/UngradedModules.jl]: @AlexD97 Let us discuss this tomorrow in gathertown or via phone.

**Comment by benlorenz** [comment]
`setup-julia` action is failing because some AWS credits renewal for the julia organization didn't happen as promised/expected and now the account is suspended.
```
$ wget https://julialang-s3.julialang.org/bin/versions.json
--2021-12-15 22:41:41--  https://julialang-s3.julialang.org/bin/versions.json
Resolving julialang-s3.julialang.org... 199.232.190.49, 2a04:4e42:6f::561
Connecting to julialang-s3.julialang.org|199.232.190.49|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2021-12-15 22:41:43 ERROR 403: Forbidden.
```

**Review by fieker — APPROVED** [review]
